src/Workspaces/Core/Portable/CodeStyle/NotificationOption.cs

```diff
@@ -9,28 +9,23 @@
 using WorkspacesResources = Microsoft.CodeAnalysis.CodeStyleResources;
 #endif
 
-#if CODE_STYLE
-namespace Microsoft.CodeAnalysis.Internal.Options
-#else
 namespace Microsoft.CodeAnalysis.CodeStyle
-#endif
 {
-    /// <summary>
-    /// Offers different notification styles for enforcing
-    /// a code style. Under the hood, it simply maps to <see cref="DiagnosticSeverity"/>
-    /// </summary>
-    /// <remarks>
-    /// This also supports various properties for databinding.
-    /// </remarks>
-    /// <completionlist cref="NotificationOption"/>
+    /// <inheritdoc cref="NotificationOption2"/>
     public class NotificationOption
```

Comment:
```suggestion
    public sealed class NotificationOption
```

Reply:
Avoiding public API changes in this PR. Tracked with https://github.com/dotnet/roslyn/issues/42550#issuecomment-600921671